app/views/notifications/index.html.erb

```diff
@@ -19,25 +19,42 @@
 
       <% elsif @notifications.present? && current_user.user_type == "tasker" %>
         <div class="notification-feed col s8">
+
           <% @notifications.each do |object| %>
             <div class="viewed feed">
               <img src="<%= object.sender.image_url %>" alt="" />
               <p class="title"> <%= object.message %> </p>
-              <a href="#modal1" onclick="modal_object(<%= object.sender.to_json %>, <%= object.id %>)" class="btn info_btn modal-trigger"> view information</a>
+              <%if object.notifiable_type == 'Quote'%>
```

Comment:
It's preferred to compare string in one case (either upcase or downcase). It's usually downcase. Compare `object.notifiable_type` against `quote` instead.